typescript/create-onchain-agent/templates/actionProvider/actionProvider.njk

```diff
@@ -0,0 +1,37 @@
+import { CustomActionProvider, {{ walletProvider }}, Network } from "@coinbase/agentkit";
+import { {{ schemaName }} } from "./schemas";
+import { z } from "zod";
+
+/**
+ * Action provider for {{ className }} operations.
+ */
+export class {{ className }} extends CustomActionProvider<{{ walletProvider }}> {
```

Comment:
I'd like to encourage usage of [customActionProvider](https://github.com/coinbase/agentkit/blob/main/typescript/agentkit/src/action-providers/customActionProvider.ts) since it's a nicer devx

Reply:
Ah interesting, yes I was not aware of this style. I'll do the refactor